Index: TopCPTools/Root/TopToolStore.cxx
===================================================================
--- TopCPTools/Root/TopToolStore.cxx    (revision 795479)
+++ TopCPTools/Root/TopToolStore.cxx    (working copy)
@@ -382,7 +382,7 @@
                    "Failed to set ESModel for " + egamma_calib_name);
       } else if (m_release_series == 24) {
         top::check(asg::setProperty(egammaCalibrationAndSmearingTool,
-                                    "ESModel", "es2016PRE"),
+                                    "ESModel", "es2016data_mc15c"),
                    "Failed to set ESModel for " + egamma_calib_name);
       }
       top::check(asg::setProperty(egammaCalibrationAndSmearingTool,
@@ -1255,13 +1255,19 @@
     if (m_release_series == 23) {
       if (largeRJES_config.find("UJ2015_") != 0)
         largeRJES_config.insert(0, "UJ2015_");
+    } else if (m_release_series == 24) {
+      if (largeRJES_config.find("UJ2016_") != 0)
+        largeRJES_config.insert(0, "UJ2016_");
+    } else {
+      ATH_MSG_ERROR("Unknown release series");
+      return StatusCode::FAILURE;
     }
 
     std::string conference;
     if (m_release_series == 23) {
-      conference = "Moriond2016";
+      conference = "UJ_2015/Moriond2016";
     } else if (m_release_series == 24) {
-      conference = "ICHEP2016";
+      conference = "UJ_2016/Moriond2017";
     } else {
       ATH_MSG_ERROR("Unknown release series");
       return StatusCode::FAILURE;
@@ -1273,17 +1279,17 @@
     m_jetUncertaintiesToolLargeR_strong
       = setupJetUncertaintiesTool("JetUncertaintiesToolLargeR_Strong",
                                   jetCalibrationNameLargeR, MC_type,
-                                  "UJ_2015/" + conference
+                                  conference
                                   + "/"+largeRJES_config+"_strong.config");
     m_jetUncertaintiesToolLargeR_medium
       = setupJetUncertaintiesTool("JetUncertaintiesToolLargeR_Medium",
                                   jetCalibrationNameLargeR, MC_type,
-                                  "UJ_2015/" + conference
+                                  conference
                                   +"/"+largeRJES_config+"_medium.config");
     m_jetUncertaintiesToolLargeR_weak
       = setupJetUncertaintiesTool("JetUncertaintiesToolLargeR_Weak",
                                   jetCalibrationNameLargeR, MC_type,
-                                  "UJ_2015/" + conference
+                                  conference
                                   + "/"+largeRJES_config+"_weak.config");
     return StatusCode::SUCCESS;
   }
