Index: Root/EventSaverFlatNtuple.cxx
===================================================================
--- Root/EventSaverFlatNtuple.cxx	(revision 803877)
+++ Root/EventSaverFlatNtuple.cxx	(working copy)
@@ -363,90 +363,90 @@
                     systematicTree->makeOutputVariable(m_weight_pileup_UP	, "weight_pileup_UP");
                     systematicTree->makeOutputVariable(m_weight_pileup_DOWN	, "weight_pileup_DOWN");
 
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_EL_SF_Trigger_UP,   "weight_leptonSF_EL_SF_Trigger_UP");
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_EL_SF_Trigger_DOWN, "weight_leptonSF_EL_SF_Trigger_DOWN");
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_EL_SF_Reco_UP,      "weight_leptonSF_EL_SF_Reco_UP");
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_EL_SF_Reco_DOWN,    "weight_leptonSF_EL_SF_Reco_DOWN");
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_EL_SF_ID_UP,        "weight_leptonSF_EL_SF_ID_UP");
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_EL_SF_ID_DOWN,      "weight_leptonSF_EL_SF_ID_DOWN");
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_EL_SF_Isol_UP,      "weight_leptonSF_EL_SF_Isol_UP");
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_EL_SF_Isol_DOWN,    "weight_leptonSF_EL_SF_Isol_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_EL_SF_Trigger_UP,   "weight_leptonSF_EL_SF_Trigger_UP");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_EL_SF_Trigger_DOWN, "weight_leptonSF_EL_SF_Trigger_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_EL_SF_Reco_UP,      "weight_leptonSF_EL_SF_Reco_UP");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_EL_SF_Reco_DOWN,    "weight_leptonSF_EL_SF_Reco_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_EL_SF_ID_UP,        "weight_leptonSF_EL_SF_ID_UP");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_EL_SF_ID_DOWN,      "weight_leptonSF_EL_SF_ID_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_EL_SF_Isol_UP,      "weight_leptonSF_EL_SF_Isol_UP");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_EL_SF_Isol_DOWN,    "weight_leptonSF_EL_SF_Isol_DOWN");
 
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_Trigger_STAT_UP,   "weight_leptonSF_MU_SF_Trigger_STAT_UP");
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_Trigger_STAT_DOWN, "weight_leptonSF_MU_SF_Trigger_STAT_DOWN");
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_Trigger_SYST_UP,   "weight_leptonSF_MU_SF_Trigger_SYST_UP");
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_Trigger_SYST_DOWN, "weight_leptonSF_MU_SF_Trigger_SYST_DOWN");
-                    // Muon ID SF systematics (regular)
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_ID_STAT_UP,   "weight_leptonSF_MU_SF_ID_STAT_UP");
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_ID_STAT_DOWN, "weight_leptonSF_MU_SF_ID_STAT_DOWN");
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_ID_SYST_UP,   "weight_leptonSF_MU_SF_ID_SYST_UP");
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_ID_SYST_DOWN, "weight_leptonSF_MU_SF_ID_SYST_DOWN");
-                    // Muon ID SF systematics (low pt)
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_ID_STAT_LOWPT_UP,   "weight_leptonSF_MU_SF_ID_STAT_LOWPT_UP");
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_ID_STAT_LOWPT_DOWN, "weight_leptonSF_MU_SF_ID_STAT_LOWPT_DOWN");
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_ID_SYST_LOWPT_UP,   "weight_leptonSF_MU_SF_ID_SYST_LOWPT_UP");
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_ID_SYST_LOWPT_DOWN, "weight_leptonSF_MU_SF_ID_SYST_LOWPT_DOWN");
-                    // Muon isolation SF systematics
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_Isol_STAT_UP,      "weight_leptonSF_MU_SF_Isol_STAT_UP");
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_Isol_STAT_DOWN,    "weight_leptonSF_MU_SF_Isol_STAT_DOWN");
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_Isol_SYST_UP,      "weight_leptonSF_MU_SF_Isol_SYST_UP");
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_Isol_SYST_DOWN,    "weight_leptonSF_MU_SF_Isol_SYST_DOWN");
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_TTVA_STAT_UP,      "weight_leptonSF_MU_SF_TTVA_STAT_UP");
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_TTVA_STAT_DOWN,    "weight_leptonSF_MU_SF_TTVA_STAT_DOWN");
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_TTVA_SYST_UP,      "weight_leptonSF_MU_SF_TTVA_SYST_UP");
-                    systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_TTVA_SYST_DOWN,    "weight_leptonSF_MU_SF_TTVA_SYST_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_Trigger_STAT_UP,   "weight_leptonSF_MU_SF_Trigger_STAT_UP");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_Trigger_STAT_DOWN, "weight_leptonSF_MU_SF_Trigger_STAT_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_Trigger_SYST_UP,   "weight_leptonSF_MU_SF_Trigger_SYST_UP");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_Trigger_SYST_DOWN, "weight_leptonSF_MU_SF_Trigger_SYST_DOWN");
+                    // // Muon ID SF systematics (regular)
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_ID_STAT_UP,   "weight_leptonSF_MU_SF_ID_STAT_UP");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_ID_STAT_DOWN, "weight_leptonSF_MU_SF_ID_STAT_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_ID_SYST_UP,   "weight_leptonSF_MU_SF_ID_SYST_UP");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_ID_SYST_DOWN, "weight_leptonSF_MU_SF_ID_SYST_DOWN");
+                    // // Muon ID SF systematics (low pt)
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_ID_STAT_LOWPT_UP,   "weight_leptonSF_MU_SF_ID_STAT_LOWPT_UP");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_ID_STAT_LOWPT_DOWN, "weight_leptonSF_MU_SF_ID_STAT_LOWPT_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_ID_SYST_LOWPT_UP,   "weight_leptonSF_MU_SF_ID_SYST_LOWPT_UP");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_ID_SYST_LOWPT_DOWN, "weight_leptonSF_MU_SF_ID_SYST_LOWPT_DOWN");
+                    // // Muon isolation SF systematics
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_Isol_STAT_UP,      "weight_leptonSF_MU_SF_Isol_STAT_UP");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_Isol_STAT_DOWN,    "weight_leptonSF_MU_SF_Isol_STAT_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_Isol_SYST_UP,      "weight_leptonSF_MU_SF_Isol_SYST_UP");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_Isol_SYST_DOWN,    "weight_leptonSF_MU_SF_Isol_SYST_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_TTVA_STAT_UP,      "weight_leptonSF_MU_SF_TTVA_STAT_UP");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_TTVA_STAT_DOWN,    "weight_leptonSF_MU_SF_TTVA_STAT_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_TTVA_SYST_UP,      "weight_leptonSF_MU_SF_TTVA_SYST_UP");
+                    // systematicTree->makeOutputVariable(m_weight_leptonSF_MU_SF_TTVA_SYST_DOWN,    "weight_leptonSF_MU_SF_TTVA_SYST_DOWN");
 
 
                     // write also out the individual components:
 
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_Trigger,         "weight_indiv_SF_EL_Trigger");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_Trigger_UP,      "weight_indiv_SF_EL_Trigger_UP");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_Trigger_DOWN,    "weight_indiv_SF_EL_Trigger_DOWN");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_Reco,            "weight_indiv_SF_EL_Reco");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_Reco_UP,         "weight_indiv_SF_EL_Reco_UP");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_Reco_DOWN,       "weight_indiv_SF_EL_Reco_DOWN");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_ID,              "weight_indiv_SF_EL_ID");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_ID_UP,           "weight_indiv_SF_EL_ID_UP");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_ID_DOWN,         "weight_indiv_SF_EL_ID_DOWN");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_Isol,            "weight_indiv_SF_EL_Isol");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_Isol_UP,         "weight_indiv_SF_EL_Isol_UP");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_Isol_DOWN,       "weight_indiv_SF_EL_Isol_DOWN");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_ChargeID,        "weight_indiv_SF_EL_ChargeID");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_ChargeID_UP,     "weight_indiv_SF_EL_ChargeID_UP");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_ChargeID_DOWN,   "weight_indiv_SF_EL_ChargeID_DOWN");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_ChargeMisID,     "weight_indiv_SF_EL_ChargeMisID");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_ChargeMisID_STAT_UP,  "weight_indiv_SF_EL_ChargeMisID_STAT_UP");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_ChargeMisID_STAT_DOWN,"weight_indiv_SF_EL_ChargeMisID_STAT_DOWN");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_ChargeMisID_SYST_UP,  "weight_indiv_SF_EL_ChargeMisID_SYST_UP");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_ChargeMisID_SYST_DOWN,"weight_indiv_SF_EL_ChargeMisID_SYST_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_Trigger,         "weight_indiv_SF_EL_Trigger");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_Trigger_UP,      "weight_indiv_SF_EL_Trigger_UP");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_Trigger_DOWN,    "weight_indiv_SF_EL_Trigger_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_Reco,            "weight_indiv_SF_EL_Reco");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_Reco_UP,         "weight_indiv_SF_EL_Reco_UP");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_Reco_DOWN,       "weight_indiv_SF_EL_Reco_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_ID,              "weight_indiv_SF_EL_ID");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_ID_UP,           "weight_indiv_SF_EL_ID_UP");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_ID_DOWN,         "weight_indiv_SF_EL_ID_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_Isol,            "weight_indiv_SF_EL_Isol");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_Isol_UP,         "weight_indiv_SF_EL_Isol_UP");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_Isol_DOWN,       "weight_indiv_SF_EL_Isol_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_ChargeID,        "weight_indiv_SF_EL_ChargeID");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_ChargeID_UP,     "weight_indiv_SF_EL_ChargeID_UP");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_ChargeID_DOWN,   "weight_indiv_SF_EL_ChargeID_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_ChargeMisID,     "weight_indiv_SF_EL_ChargeMisID");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_ChargeMisID_STAT_UP,  "weight_indiv_SF_EL_ChargeMisID_STAT_UP");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_ChargeMisID_STAT_DOWN,"weight_indiv_SF_EL_ChargeMisID_STAT_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_ChargeMisID_SYST_UP,  "weight_indiv_SF_EL_ChargeMisID_SYST_UP");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_EL_ChargeMisID_SYST_DOWN,"weight_indiv_SF_EL_ChargeMisID_SYST_DOWN");
 
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_Trigger,         "weight_indiv_SF_MU_Trigger");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_Trigger_STAT_UP,      "weight_indiv_SF_MU_Trigger_STAT_UP");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_Trigger_STAT_DOWN,    "weight_indiv_SF_MU_Trigger_STAT_DOWN");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_Trigger_SYST_UP,      "weight_indiv_SF_MU_Trigger_SYST_UP");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_Trigger_SYST_DOWN,    "weight_indiv_SF_MU_Trigger_SYST_DOWN");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_ID,              "weight_indiv_SF_MU_ID");
-                    // Muon ID SF systematics (regular)
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_ID_STAT_UP,      "weight_indiv_SF_MU_ID_STAT_UP");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_ID_STAT_DOWN,    "weight_indiv_SF_MU_ID_STAT_DOWN");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_ID_SYST_UP,      "weight_indiv_SF_MU_ID_SYST_UP");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_ID_SYST_DOWN,    "weight_indiv_SF_MU_ID_SYST_DOWN");
-                    // Muon ID SF systematics (low pt)
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_ID_STAT_LOWPT_UP,      "weight_indiv_SF_MU_ID_STAT_LOWPT_UP");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_ID_STAT_LOWPT_DOWN,    "weight_indiv_SF_MU_ID_STAT_LOWPT_DOWN");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_ID_SYST_LOWPT_UP,      "weight_indiv_SF_MU_ID_SYST_LOWPT_UP");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_ID_SYST_LOWPT_DOWN,    "weight_indiv_SF_MU_ID_SYST_LOWPT_DOWN");
-                    // Muon isolation SF systematics
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_Isol,            "weight_indiv_SF_MU_Isol");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_Isol_STAT_UP,    "weight_indiv_SF_MU_Isol_STAT_UP");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_Isol_STAT_DOWN,  "weight_indiv_SF_MU_Isol_STAT_DOWN");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_Isol_SYST_UP,    "weight_indiv_SF_MU_Isol_SYST_UP");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_Isol_SYST_DOWN,  "weight_indiv_SF_MU_Isol_SYST_DOWN");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_TTVA,            "weight_indiv_SF_MU_TTVA");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_TTVA_STAT_UP,    "weight_indiv_SF_MU_TTVA_STAT_UP");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_TTVA_STAT_DOWN,  "weight_indiv_SF_MU_TTVA_STAT_DOWN");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_TTVA_SYST_UP,    "weight_indiv_SF_MU_TTVA_SYST_UP");
-                    systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_TTVA_SYST_DOWN,  "weight_indiv_SF_MU_TTVA_SYST_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_Trigger,         "weight_indiv_SF_MU_Trigger");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_Trigger_STAT_UP,      "weight_indiv_SF_MU_Trigger_STAT_UP");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_Trigger_STAT_DOWN,    "weight_indiv_SF_MU_Trigger_STAT_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_Trigger_SYST_UP,      "weight_indiv_SF_MU_Trigger_SYST_UP");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_Trigger_SYST_DOWN,    "weight_indiv_SF_MU_Trigger_SYST_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_ID,              "weight_indiv_SF_MU_ID");
+                    // // Muon ID SF systematics (regular)
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_ID_STAT_UP,      "weight_indiv_SF_MU_ID_STAT_UP");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_ID_STAT_DOWN,    "weight_indiv_SF_MU_ID_STAT_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_ID_SYST_UP,      "weight_indiv_SF_MU_ID_SYST_UP");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_ID_SYST_DOWN,    "weight_indiv_SF_MU_ID_SYST_DOWN");
+                    // // Muon ID SF systematics (low pt)
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_ID_STAT_LOWPT_UP,      "weight_indiv_SF_MU_ID_STAT_LOWPT_UP");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_ID_STAT_LOWPT_DOWN,    "weight_indiv_SF_MU_ID_STAT_LOWPT_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_ID_SYST_LOWPT_UP,      "weight_indiv_SF_MU_ID_SYST_LOWPT_UP");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_ID_SYST_LOWPT_DOWN,    "weight_indiv_SF_MU_ID_SYST_LOWPT_DOWN");
+                    // // Muon isolation SF systematics
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_Isol,            "weight_indiv_SF_MU_Isol");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_Isol_STAT_UP,    "weight_indiv_SF_MU_Isol_STAT_UP");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_Isol_STAT_DOWN,  "weight_indiv_SF_MU_Isol_STAT_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_Isol_SYST_UP,    "weight_indiv_SF_MU_Isol_SYST_UP");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_Isol_SYST_DOWN,  "weight_indiv_SF_MU_Isol_SYST_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_TTVA,            "weight_indiv_SF_MU_TTVA");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_TTVA_STAT_UP,    "weight_indiv_SF_MU_TTVA_STAT_UP");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_TTVA_STAT_DOWN,  "weight_indiv_SF_MU_TTVA_STAT_DOWN");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_TTVA_SYST_UP,    "weight_indiv_SF_MU_TTVA_SYST_UP");
+                    // systematicTree->makeOutputVariable(m_weight_indiv_SF_MU_TTVA_SYST_DOWN,  "weight_indiv_SF_MU_TTVA_SYST_DOWN");
 
                     if (m_config->useTaus()) {
                       systematicTree->makeOutputVariable(m_weight_tauSF_ELEOLR_UP,
@@ -492,18 +492,18 @@
                       // skip uncalibrated though available WPs
                       if (std::find(m_config->bTagWP_calibrated().begin(), m_config->bTagWP_calibrated().end(), tagWP) == m_config->bTagWP_calibrated().end()) continue;
                       // up
-                      systematicTree->makeOutputVariable(m_weight_bTagSF_eigen_B_up[tagWP], "weight_bTagSF_"+shortBtagWP(tagWP)+"_eigenvars_B_up");
-                      systematicTree->makeOutputVariable(m_weight_bTagSF_eigen_C_up[tagWP], "weight_bTagSF_"+shortBtagWP(tagWP)+"_eigenvars_C_up");
-                      systematicTree->makeOutputVariable(m_weight_bTagSF_eigen_Light_up[tagWP], "weight_bTagSF_"+shortBtagWP(tagWP)+"_eigenvars_Light_up");
-                      // down
-                      systematicTree->makeOutputVariable(m_weight_bTagSF_eigen_B_down[tagWP], "weight_bTagSF_"+shortBtagWP(tagWP)+"_eigenvars_B_down");
-                      systematicTree->makeOutputVariable(m_weight_bTagSF_eigen_C_down[tagWP], "weight_bTagSF_"+shortBtagWP(tagWP)+"_eigenvars_C_down");
-                      systematicTree->makeOutputVariable(m_weight_bTagSF_eigen_Light_down[tagWP], "weight_bTagSF_"+shortBtagWP(tagWP)+"_eigenvars_Light_down");
+                      // systematicTree->makeOutputVariable(m_weight_bTagSF_eigen_B_up[tagWP], "weight_bTagSF_"+shortBtagWP(tagWP)+"_eigenvars_B_up");
+                      // systematicTree->makeOutputVariable(m_weight_bTagSF_eigen_C_up[tagWP], "weight_bTagSF_"+shortBtagWP(tagWP)+"_eigenvars_C_up");
+                      // systematicTree->makeOutputVariable(m_weight_bTagSF_eigen_Light_up[tagWP], "weight_bTagSF_"+shortBtagWP(tagWP)+"_eigenvars_Light_up");
+                      // // down
+                      // systematicTree->makeOutputVariable(m_weight_bTagSF_eigen_B_down[tagWP], "weight_bTagSF_"+shortBtagWP(tagWP)+"_eigenvars_B_down");
+                      // systematicTree->makeOutputVariable(m_weight_bTagSF_eigen_C_down[tagWP], "weight_bTagSF_"+shortBtagWP(tagWP)+"_eigenvars_C_down");
+                      // systematicTree->makeOutputVariable(m_weight_bTagSF_eigen_Light_down[tagWP], "weight_bTagSF_"+shortBtagWP(tagWP)+"_eigenvars_Light_down");
                       // named systematics
-                      for (auto name : m_config->btagging_namedSysts(tagWP)) {
-                        systematicTree->makeOutputVariable(m_weight_bTagSF_named_up[tagWP][name], "weight_bTagSF_"+shortBtagWP(tagWP)+"_"+betterBtagNamedSyst(name)+"_up" );
-                        systematicTree->makeOutputVariable(m_weight_bTagSF_named_down[tagWP][name], "weight_bTagSF_"+shortBtagWP(tagWP)+"_"+betterBtagNamedSyst(name)+"_down" );
-                      }
+                      // for (auto name : m_config->btagging_namedSysts(tagWP)) {
+                      //   systematicTree->makeOutputVariable(m_weight_bTagSF_named_up[tagWP][name], "weight_bTagSF_"+shortBtagWP(tagWP)+"_"+betterBtagNamedSyst(name)+"_up" );
+                      //   systematicTree->makeOutputVariable(m_weight_bTagSF_named_down[tagWP][name], "weight_bTagSF_"+shortBtagWP(tagWP)+"_"+betterBtagNamedSyst(name)+"_down" );
+                      // }
                     }
                     if( m_config->useTrackJets() ) {
                       for( auto& tagWP : m_config -> bTagWP_available_trkJet()) {
@@ -644,12 +644,12 @@
 
             //large-R jets
             if (m_config->useLargeRJets()) {
-                systematicTree->makeOutputVariable(m_ljet_pt,   "ljet_pt");
-                systematicTree->makeOutputVariable(m_ljet_eta,  "ljet_eta");
-                systematicTree->makeOutputVariable(m_ljet_phi,  "ljet_phi");
-                systematicTree->makeOutputVariable(m_ljet_e,    "ljet_e");
-                systematicTree->makeOutputVariable(m_ljet_m,    "ljet_m");
-                systematicTree->makeOutputVariable(m_ljet_sd12, "ljet_sd12");
+                // systematicTree->makeOutputVariable(m_ljet_pt,   "ljet_pt");
+                // systematicTree->makeOutputVariable(m_ljet_eta,  "ljet_eta");
+                // systematicTree->makeOutputVariable(m_ljet_phi,  "ljet_phi");
+                // systematicTree->makeOutputVariable(m_ljet_e,    "ljet_e");
+                // systematicTree->makeOutputVariable(m_ljet_m,    "ljet_m");
+                // systematicTree->makeOutputVariable(m_ljet_sd12, "ljet_sd12");
             }
 
             //track jets
@@ -918,13 +918,13 @@
 
         //jets
         if ( m_config->useTruthJets() ){
-            m_particleLevelTreeManager->makeOutputVariable(m_jet_pt, "jet_pt");
-            m_particleLevelTreeManager->makeOutputVariable(m_jet_eta, "jet_eta");
-            m_particleLevelTreeManager->makeOutputVariable(m_jet_phi, "jet_phi");
-            m_particleLevelTreeManager->makeOutputVariable(m_jet_e, "jet_e");
+            // m_particleLevelTreeManager->makeOutputVariable(m_jet_pt, "jet_pt");
+            // m_particleLevelTreeManager->makeOutputVariable(m_jet_eta, "jet_eta");
+            // m_particleLevelTreeManager->makeOutputVariable(m_jet_phi, "jet_phi");
+            // m_particleLevelTreeManager->makeOutputVariable(m_jet_e, "jet_e");
 
-            m_particleLevelTreeManager->makeOutputVariable(m_jet_Ghosts_BHadron_Final_Count, "jet_nGhosts_bHadron");
-            m_particleLevelTreeManager->makeOutputVariable(m_jet_Ghosts_CHadron_Final_Count, "jet_nGhosts_cHadron");
+            // m_particleLevelTreeManager->makeOutputVariable(m_jet_Ghosts_BHadron_Final_Count, "jet_nGhosts_bHadron");
+            // m_particleLevelTreeManager->makeOutputVariable(m_jet_Ghosts_CHadron_Final_Count, "jet_nGhosts_cHadron");
         }
 
         //large R jets
